cpp/proto/arcticc/pb2/s3_storage.proto

```diff
@@ -27,4 +29,6 @@ message Config {
     string ca_cert_dir = 15;
     // Whether to leave the prefix unchanged, or to translate the periods in it.
     bool use_raw_prefix = 16;
+    arcticc.pb2.storage_pb2.AWSAuthMethod aws_auth = 17; //default to be 0, DISABLED
```

Comment:
I'm very worried about adding anything to these protobufs after all the issues recently with the `https` flag.

There is actually no reason for this to go in the protobuf at all is there? We never intend to serialize it?

We will need extensive compatibility testing (similar to https://github.com/man-group/ArcticDB/pull/1862 ) for this change either way.